content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -56,18 +56,17 @@ route, we recommend you add IP route(s) so Kubernetes cluster addresses go via t
 As a requirement for your Linux Node's iptables to correctly see bridged traffic, you should ensure `net.bridge.bridge-nf-call-iptables` is set to 1 in your `sysctl` config, e.g.
 
 ```bash
-cat <<EOF > /etc/sysctl.d/k8s.conf
+cat <<EOF | sudo tee /etc/sysctl.d/k8s.conf
 net.bridge.bridge-nf-call-ip6tables = 1
 net.bridge.bridge-nf-call-iptables = 1
```

Comment:
I also needed to add `net.ipv4.ip_forward = 1` but this is maybe `containerd` specific.

Reply:
a new sentence bellow could indicate that CR might have specific requirements linking to https://kubernetes.io/docs/setup/production-environment/container-runtimes/